src/components/PopupWithForm.js

```diff
@@ -6,19 +6,26 @@ export default class PopupwithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
     this._form = this._popupElement.querySelector(".modal__form");
     this._submitBtn = this._popupElement.querySelector(".modal__save");
+    this._inputsArr = Array.from(this._form.querySelectorAll(".modal__input"));
   }
 
   _getInputValues() {
-    const inputsArr = Array.from(this._form.querySelectorAll(".modal__input"));
-
     const inputValues = {};
 
-    inputsArr.forEach((input) => {
+    this._inputsArr.forEach((input) => {
       inputValues[input.name] = input.value;
     });
     return inputValues;
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`

Reply:
I know this is only "could be improved", but I am confused on what to change here.  

Am I adding setInputValues along with _getInputValues? 

Also, is it meant to replace index.js lines 223 and 224?
>  profileNameInput.value = name;
>  profileDescriptionInput.value = about;

Reply:
> Am I adding setInputValues along with _getInputValues?

Yes. 

2 different methods

One for getting the varlues, the other one is for **setting** them input the inputs

Reply:
> Also, is it meant to replace index.js lines 223 and 224?

Right